lib/ProductOpener/Food.pm

```diff
@@ -780,36 +780,44 @@ carnitine-
 # fr_synonyms is used to parse plain text nutrition facts value
 
 %Nutriments = (
-
 alcohol	=> {
-	fr => "Alcool",
-	en => "Alcohol",
-	es => "Alcohol",
-	ar=> "الكحوليات",
-	unit => '% vol',
-	it => "Alcol",
-	pt => "Álcool",
-	de => "Alkohol",
-	he => "אלכוהול",
-	ga => "Alcól",
+	ar => "الكحوليات",
+	bg => "Алкохол",
+	cs => "Alkohol",
 	da => "Alkohol",
+	de => "Alkohol",
 	el => "Αλκοόλη",
-	fi => "Alkoholi",
-	nl => "Alcohol",
-	nl_be => "Alcohol",
-	sv => "Alkohol",
-	lv => "spirts",
-	cs => "Alkohol",
+	en => "Alcohol",
+	es => "Alcohol",
 	et => "Alkohol",
+	fa => "الکل",
+	fi => "Alkoholi",
+	fr => "Alcool",
+	ga => "Alcól",
+	he => "אלכוהול",
 	hu => "Alkohol",
-	pl => "Alkohol",
-	sl => "Alkohol",
+	it => "Alcol",
+	ja => "アルコール",
 	lt => "Alkoholis",
+	lv => "spirts",
 	mt => "Alkoħol",
-	sk => "Alkohol",
+	nl => "Alcohol",
+	nl_be => "Alcohol",
+	nb => "Alkohol",
+	pl => "Alkohol",
+	pt => "Álcool",
 	ro => "Alcool",
-	bg => "Алкохол",
+	rs => "Alkohol",
+	ru => "Алкоголь",
+	sk => "Alkohol",
+	sl => "Alkohol",
+	sv => "Alkohol",
+	tr => "Alkol",
 	zh => "酒精度",
+	zh-CN => "酒精度",
```

Comment:
zh-CN is not valid perl and it will break, put either zh_cn or "zh-CN". Those won't be used at this time.